Userland/Libraries/LibGfx/Color.cpp

```diff
@@ -418,6 +413,16 @@ Color Color::from_xyz50(float x, float y, float z, float alpha)
         clamp(lroundf(alpha * 255.f), 0, 255));
 }
 
+Color Color::from_xyz50(float x, float y, float z, float alpha)
+{
+    // See commit description for these values
+    float red = 3.13397926 * x - 1.61689519 * y - 0.49070587 * z;
+    float green = -0.97840009 * x + 1.91589112 * y + 0.03339256 * z;
+    float blue = 0.07200357 * x - 0.22897505 * y + 1.40517398 * z;
```

Comment:
For both this and the next commit: is there a reason why we do these with doubles instead of floats? The OKLab conversion also uses floats for example; wondering if there is a quantifiable benefit to it.

Reply:
No particular reason, but I haven't tried with `float` either. Do you want me to change it?

Reply:
Meh, no strong feelings about it either way - when you implicitly cast from `float` to `double` or vice versa it suggests some form of precision issue, but I don't think that's the case here. Keeping everything as `float` generates simpler/fewer instructions but nothing that we could measure in practical situations.

Reply:
For reference, the CSS types use `double`s internally. So, we could use `double`s the whole time too, though I'm guessing the difference wouldn't be observable.